### PR TITLE
Merge helloworld stats and trace example into one

### PR DESCRIPTION
**Pull request by rakyll**
The merged example is required by the website.

### CODE REVIEW

File: examples/helloworld/main.go [hunk 1]
```diff
@@ -20,59 +20,88 @@ import (
...
 
+	"go.opencensus.io/examples/exporter"
 	"go.opencensus.io/stats"
 	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+	"go.opencensus.io/trace"
+)
+
+var (
+	// frontendKey allows us breakdown the recorded data
```
Comment (odeke-em) [inline examples/helloworld/main.go]: s/allows us breakdown/allows us to breakdown/g

Comment (rakyll) [reply examples/helloworld/main.go]: Done

**Comment by rakyll** [comment]
All addressed.

**Comment by odeke-em** [comment]
Great! LGTM, let's ship it!

**Pull request merged**